src/main.js

```diff
@@ -5,15 +5,32 @@ var secInput = document.querySelector('#seconds-input');
 var descInput = document.querySelector('#description-input');
 var timerButton = document.querySelector('.timer');
 var timerInput = document.querySelector('h3');
+var completeActiveDisplay = document.querySelector('.completed-activity-display');
```

Comment:
Excited to look over our variables and determine which ones can be declared within the functions and which need to remain global during our refactor review!